source/linux/Foundational_Components_Multimedia_wave5.rst

```diff
@@ -638,6 +640,35 @@ To pipe the output from ffmpeg to ffplay with hardware acceleration, the followi
 
    ffmpeg -re -codec:v hevc_v4l2m2m -i input.h265 -fps_mode passthrough -f rawvideo - | ffplay -framerate 30 -video_size 1920x1080 -f rawvideo -autoexit -
 
+FFMPEG also has support to use V4L2 HW accelerators for video encoding. The following command is an example for encoding a raw video by using the Wave5 IP:
+
+.. code-block:: console
+
+    ffmpeg -f rawvideo -pix_fmt yuv420p -s:v 352x288 -r 30 -i <input_video>.yuv -c:v h264_v4l2m2m <output_video>.h264 -y -nostdin
```

Comment:
Ah, https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace